examples/tests/valid/function_call_variadic_max_arity.test.w

```diff
@@ -0,0 +1,7 @@
+bring cloud;
+
+let func = (n: num, b: bool, ...events: Array<Json>) => {
+  log("hello");
+};
+
+func(1, true, "a", "b", "c", "d"); // variadic args should count as 1 in arity check
```

Comment:
Since this is a valid test, lets actually verify it works
```suggestion
let func = (n: num, b: bool, ...events: Array<Json>) => {
  assert(events.at(-1) == "d");
};

func(1, true, "a", "b", "c", "d"); // variadic args should count as 1 in arity check
```